tests/conftest.py

```diff
@@ -13,6 +13,11 @@ def pytest_configure(config):
         "program_trace: mark the C/C++ source file to be automatically compiled, instrumented, and run for the test",
     )
 
+    config.addinivalue_line(
```

Comment:
I think this shouldn't be needed anymore.

Reply:
Correct. Will remove.

Reply:
Done.